imessage/mac-nosip/nosip.go

```diff
@@ -57,6 +58,7 @@ func NewMacNoSIPConnector(bridge imessage.Bridge) (imessage.API, error) {
 		log:                 logger,
 		procLog:             processLogger,
 		printPayloadContent: bridge.GetConnectorConfig().LogIPCPayloads,
+		pingInterval:        time.Duration(bridge.GetConnectorConfig().PingInterval),
```

Comment:
currently it defaults to 15 nanoseconds

```suggestion
		pingInterval:        time.Duration(bridge.GetConnectorConfig().PingInterval) * time.Second,
```

Reply:
Nope, multiplied to seconds in pingLoop on line 109. Would you rather it be fully configurable rather than fixed to seconds?

Reply:
Oh good point, probably better to move the multiplication from the ping loop to here, otherwise it's confusing